x-pack/test_serverless/api_integration/test_suites/common/encrypted_saved_objects.ts

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
__question:__ Btw, what does `common` part of the path means here? Should our tests be inside `common` or adjacent to it?

Reply:
Or is it what we call `platform`? Cause `common` sounds like common test utils or something. cc @pheyos

Reply:
@azasypkin `common` in serverless means these tests are run in all three projects. You can read more about it in the serverless test readme: https://github.com/elastic/kibana/blob/main/x-pack/test_serverless/README.md#common-tests. Let me know if you have any additional questions.

Reply:
Got it, thanks @pheyos.